tailwind.config.js

```diff
@@ -2,7 +2,19 @@
 export default {
     content: ['./src/**/*.{html,js,svelte,ts}'],
     theme: {
-        extend: {},
+        extend: {
+            colors: {
+                placeholder: '#B8B6B6',
+            },
+        },
```

Comment:
double check with Micky we actually need a custom color or if we can use tailwinds default colors